src/docs/concepts-api-sdk/rate-limiting/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Rate limiting in Marketplace API and Integration API
+slug: rate-limiting
+updated: 2022-12-12
+category: concepts-api-sdk
+ingress:
+  This article describes rate limiting in Marketplace API and
+  Integration API, as well as suggests how to handle those limits
+  gracefully.
+published: true
+---
+
+Starting in XX-XX-2023, Marketplace API and Integration API will feature
+new rate limits in dev and demo environments. In addition, Integration
+API will feature new concurrency limits in dev and demo environments
+and, eventually, in production environments.
+
+<extrainfo title="What are rate limits and concurrency limits?">
+
+**Rate limit** means a limit on the number of requests from your
+marketplace that the API can process within a time span, e.g. in one
+minute.
+
+**Concurrency limit** means a limit on the number of simultaneous
+requests from your marketplace that the API can process at any given
+moment.
+
+</extrainfo>
+
+The rate limits are different for queries (fetching data) and commands
+(modifying data). Queries are rate limited at 1 request per second (60
+requests per minute) on average. Commands are rate limited at 1 request
+per 2 seconds (30 requests per minute) on average. You can find more
+information on Flex rate limits in the [API reference](TODO).
```

Comment:
Should we mention here that the rate limit applies per IP?

Reply:
Added a mention of this!